server/controllers/index.ts

```diff
@@ -1,8 +1,8 @@
 /* istanbul ignore file */
 
 import ApplicationController from './applicationController'
-import { controllers as manageControllers } from './manage'
-import { controllers as applyControllers } from './apply'
+import { controllers as apManageControllers } from './manage'
+import { controllers as apApplyControllers } from './apply'
```

Comment:
I wondere if there's a more elegant way of namespacing the controllers? What I'd like to see is a single set of "standard" controllers in the base path, and then a way of overriding the standard controllers by adding a controller to a namespaced path, and the same with views. That said, I don't think this is a blocker given you're working on a feature branch, but something worth thinking about.